web3/manager.py

```diff
@@ -33,37 +38,36 @@ def __init__(self, web3, providers, middlewares=None):
         self.providers = providers
 
     web3 = None
-    _middlewares = None
     _providers = None
 
     @property
     def middlewares(self):
-        return self._middlewares or tuple()
+        return tuple(self.middleware_stack)
```

Comment:
I figured that with the introduction of `RequestManager.middleware_stack` we'd remove this property.  Is there a reason to keep it that I'm not seeing?

Reply:
Adds some convenience, but more on the setter side than the get. I guess whether this makes sense depends on your next comment.